extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -48,6 +48,19 @@ public function getSourceLabel() {
 		return $sourceLabels;
 	}
 
+	// TODO what happens when there are multiple sources?
```

Comment:
What happens if the same source is used twice in two different types? Let's add unit tests for those questions!